analyzers/tests/SonarAnalyzer.Test/TestCases/ArrayPassedAsParams.cs

```diff
@@ -81,25 +81,27 @@ public class Repro6894
-        MethodArray(new String[] { "1", "2" }, new String[] { "1", "2" });  // Compliant, elements in args: [System.String[], System.String[]]
-        MethodArray(new int[] { 1, 2 }, new int[] { 1, 2 });                // Compliant, elements in args: [System.Int32[], System.Int32[]]
-
-        MethodJaggedArray(new int[] { 1, 2 });                              // Compliant: jagged array [System.Object[]]
-        Method(d);                                                          // Compliant
-        Method("Hello", 2);                                                 // Compliant
+        Method(new String[] { "1", "2" });                                      // Noncompliant, elements in args: ["1", "2"]
+                                                                                // The argument given for a parameter array can be a single expression that is implicitly convertible (§10.2) to the parameter array type.
+                                                                                // In this case, the parameter array acts precisely like a value parameter.
+                                                                                // see: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/classes#14625-parameter-arrays
+        Method(new object[] { new int[] { 1, 2 } });                            // FN, elements in args: [System.Int32[]]
+        Method(new int[] { 1, 2, 3, });                                         // Compliant, Elements in args: [System.Int32[]]
+        Method(new String[] { "1", "2" }, new String[] { "1", "2" });           // Compliant, elements in args: [System.String[], System.String[]]
+        Method(new String[] { "1", "2"}, new int[] { 1, 2});                    // Compliant, elements in args: [System.String[], System.Int32[]]
+        MethodMixed(1, new String[] { "1", "2" });                              // Noncompliant
+        MethodArray(new String[] { "1", "2" }, new String[] { "1", "2" });      // Compliant, elements in args: [System.String[], System.String[]]
+        MethodArray(new int[] { 1, 2 }, new int[] { 1, 2 });                    // Compliant, elements in args: [System.Int32[], System.Int32[]]
+        MethodJaggedArray(new int[] { 1, 2 });                                  // Compliant: jagged array [System.Object[]]
+        Method(d);                                                              // Compliant
+        Method("Hello", 2);                                                     // Compliant
         string.Format(CultureInfo.InvariantCulture, "{0}.{1}", new object[] { "", new object() });
+        MethodImplicitArray(new[] { "Hello", "Hi" });                           // Noncompliant , Implicit array creation
+        Method(new [] { 1, 2, 3, });                                            // Compliant, Elements in args: [System.Int32[]]
```

Comment:
```suggestion
        Method(new[] { 1, 2, 3, });                                            // Compliant, Elements in args: [System.Int32[]]
```

Reply:
Add one more test with a multidimensional array
```suggestion
        Method(new[] { 1, 2, 3, });                                            // Compliant, Elements in args: [System.Int32[]]
        Method(new[,] { { 1, 2 } });                                            // Compliant, Elements in args: [System.Int32[,]]        
```